tests/tensor/rewriting/test_math.py

```diff
@@ -4571,3 +4571,22 @@ def test_log_kv_stabilization():
         out.eval({x: 1000.0}, mode=mode),
         -1003.2180912984705,
     )
+
+
+@pytest.mark.parametrize("shape", [(), (4, 5, 6)], ids=["scalar", "tensor"])
+def test_pow_1_rewrite(shape):
+    x = pt.tensor("x", shape=shape)
+    z = 1**x
+
+    f1 = pytensor.function([x], z, mode=get_default_mode().excluding("canonicalize"))
```

Comment:
This is wasteful. Just check the Pow is there in the original graph and is gone in the compiled function?

Or test the rewrite directly like I do here: https://github.com/pymc-devs/pytensor/pull/1178/commits/64bb608b02b4fd193c18d790c7c4be2f5aeb3846